### PR TITLE
Added index.html and styles.css

### PR DESCRIPTION
**Pull request by richardoppiyo**
This pull request is for addition of a new feature

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (richardoppiyo) [inline index.html:62]: Kindly, do not leave commented and unused code on your codebase. This makes your codebase unclean and hard to read.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (richardoppiyo) [inline css/style.css:1]: Please, consider removing a lot of unnecessary white spaces in your CSS. Leaving such white spaces leads to linter errors.
Kindly set up Github's Continuous Integration tool to help you catch such errors.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+<body>
+  
+ <header id="header">
+  <nav id="nav-bar" class="navbar navbar-light bg-light">
+    <div class="container">
+      <a class="navbar-brand" href="#">
+        <img id="header-img" src="img/microverse.png" alt="" width="30" height="24">
+      </a>
+
```
Comment (richardoppiyo) [inline index.html:21]: Kindly check your indentations.

File: babel/index.babel [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    from red to green.
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
+// PLEASE NOTE: Adding global style rules using the * selector, or by adding rules to body {..} or html {..}, or to all elements within body or html, i.e. h1 {..}, has the potential to pollute the test suite's CSS. Try adding: * { color: red }, for a quick example!
+
+// Once you have read the above messages, you can delete all comments.
```
Comment (richardoppiyo) [inline babel/index.babel:27]: Kindly do not leave commented code in your codebase. Commented code makes your codebase untidy. 😎 .But you are doing great already! 👌

File: index.html [hunk 5]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+
```
Comment (richardoppiyo) [inline index.html:6]: It is a good practice to add a viewport meta tag. Kindly add the same to your code

**Review by richardoppiyo — COMMENTED** [review]
# Changes required  ♻️ 

Kindly check my comments and make the required changes.

Once done you can submit for a re-review.

Regards 😄

File: babel/index.babel (hunk 4 above)
Comment (rotimiazeez) [reply babel/index.babel:27]: - [ ] Great job with all your review I'd suggest you add emojis to make your review more friendly 😎👍

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
Hi @richardoppiyo 🔥, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: babel/index.babel (hunk 4 above)
Comment (richardoppiyo) [reply babel/index.babel:27]: > * [ ]  Great job with all your review I'd suggest you add emojis to make your review more friendly 😎👍

Noted with thanks.
💯  😃  👍

**Review by richardoppiyo — COMMENTED** [review]
# Changes required ♻️ 

You've done a great job done so far implementing the changes 👍 

Kindly check the comments and make just a little more change 😎.

Regards.

**Review by Kingstalux — APPROVED** [review]
Hi @richardoppiyo , 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Good job implementing the changes required by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**